lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -772,24 +773,9 @@ def on_tls_relation_broken(self, _: RelationBrokenEvent):
 
     def is_tls_fully_configured(self) -> bool:
```

Comment:
I believe we can get rid of this method - and directly set / remove the `tls_configured` relation data flag from the TLS module